src/languages/dart.js

```diff
@@ -72,17 +72,28 @@ export default function(hljs) {
     hljs.C_NUMBER_MODE, STRING
   ];
 
+  var BUILT_IN_TYPES =
```

Comment:
Please just go ahead and explode this into an array with one item per line.  We need to start doing this anyways and it will clean up the logic below.

Reply:
Done